operators/Makefile

```diff
@@ -43,7 +43,7 @@ TAG ?= $(shell git rev-parse --short --verify HEAD)
 
 ifeq ($(OPERATOR_IMAGE),)
 	# we never want this empty
-	undefine OPERATOR_IMAGE
+	OPERATOR_IMAGE := $(IMG):$(VERSION)-$(TAG)
```

Comment:
Isn't it already the purpose of `OPERATOR_IMAGE ?= ` below?

Reply:
It performs the assignment only if variable is not set, but not if it's set, but empty.

Reply:
In which situation the variable is set to empty? It looks like a bug.

Reply:
This was added because:
- e2e test target sometimes needs it (if testing particular operator image) and sometimes doesn’t (if it’s supposed to build and test)
- OPERATOR_IMAGE is passed for the first case and it’s empty for the second case
- in the second case, we want use the default, so undefing for the next lines to work
- alternative was to create a separate (effectively duplicated) target, but I really wanted to avoid that

I hope that this (among other things) can be improved after working through #1528.